src/main/java/at/hannibal2/skyhanni/features/misc/MarkedPlayerManager.kt

```diff
@@ -88,7 +90,7 @@ class MarkedPlayerManager {
     @SubscribeEvent
     fun onRenderMobColored(event: RenderMobColoredEvent) {
         if (!LorenzUtils.inSkyBlock) return
-        if (!SkyHanniMod.feature.markedPlayers.highlightInWorld) return
+        if (!config.highlightInWorld) return
```

Comment:
All separate ifs here too. Why?

Reply:
Here the usage of an isEnabled function is more consistent with the rest of the code base as well.